cellprofiler/modules/loadimages.py

```diff
@@ -3398,7 +3461,6 @@ def __provide_volume(self):
 
 class LoadImagesImageProviderURL(LoadImagesImageProvider):
     '''Reference an image via a URL'''
```

Comment:
Let's not make whitespace changes on code we are not changing.

Reply:
Sorry, whitespace changes were unintentional/sloppy. Will check in future!